eve/main.yml

```diff
@@ -65,13 +65,15 @@ models:
       S3BACKEND: "mem"
       MPU_TESTING: "yes"
       S3METADATA: mongodb
+      REDIS_TEST: "true"
```

Comment:
I am worried about introducing another environment variable. Can we do without?

Reply:
will fix and use CI=true instead